package.json

```diff
@@ -17,7 +17,7 @@
     "build": "tsc --project tsconfig.build.json",
     "clean": "rm -rf dist && mkdir dist && rm -rf bin && mkdir bin",
     "lint": "tsc --noEmit --project tsconfig.json && eslint --ext .ts src --quiet",
-    "test": "LOG_LEVEL=none nyc ts-mocha -p tsconfig.json tests/**/*.spec.ts",
+    "test": "LOG_LEVEL=none MASTER_ROUTE=/master nyc ts-mocha -p tsconfig.json tests/**/*.spec.ts",
```

Comment:
I think ideally the tests would be able to stay agnostic about "runtime" details like this. So if this was something we needed to set for specific tests it should stay scoped to those particular tests.

Reply:
Good point, thanks I pushed change